src/styles/colour/index.md.njk

```diff
@@ -12,8 +12,8 @@ Always use the GOV.UK colour palette.
 ## Colour contrast
 You must make sure that the contrast ratio of text and interactive elements in
 your service meets [level AA of the Web Content Accessibility Guidelines
-(WCAG 2.0)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
-
+- (WCAG 2.0)](https://www.w3.org/TR/UNDERSTANDING-WCAG20/visual-audio-contrast-contrast.html)
```

Comment:
Hey Lauren!

Thanks for this change it looks good but there's one thing that needs fixing before we can publish this.

The `-` and `+` characters are to represent what needs to be added can be removed.

You can edit the file again by using the following link: https://github.com/LaurenMcAllister/govuk-design-system/edit/patch-2/src/styles/colour/index.md.njk?pr=%2Falphagov%2Fgovuk-design-system%2Fpull%2F763

Let us know if you need any help with this change.